assets/src/dashboard/app/router/routerProvider.js

```diff
@@ -55,6 +55,24 @@ function RouterProvider({ children, ...props }) {
     trackScreenView(currentScreen);
   }, [currentScreen]);
 
+  // Sync up WP navigation bar with our hash location.
+  useEffect(() => {
+    // Allow default WP behavior for both `page=stories-dashboard` & `page=stories-dashboard#/`
+    if (currentPath.length <= 1) {
+      return;
+    }
+
+    const WPSubmenuItems = document.querySelectorAll(
```

Comment:
Just realized WP adds the `.current` class to both the `li` and the `a`, and also adds `aria-current="page"` to the `a` element 😕 Not sure we want to do the same for consistency. Akes it a bit more complex than what I thought.

Reply:
yea I can add that in no prob

Reply:
Then let's do it 🙂 👍